ghost/staff-service/lib/emails.js

```diff
@@ -170,13 +170,50 @@ class StaffServiceEmails {
      * @returns {Promise<void>}
      */
     async notifyMilestoneReceived({milestone}) {
+        if (!milestone?.emailSentAt || (milestone?.meta && milestone?.meta?.reason)) {
+            // Do not send an email when no email was set to be sent or a reason
+            // not to send provided
+            return;
+        }
+
+        const emailPromises = [];
         const users = await this.models.User.getEmailAlertUsers('milestone-received');
+        const formattedValue = this.getFormattedAmount({currency: milestone?.currency, amount: milestone.value, maximumFractionDigits: 0});
```

Comment:
Don't know the whole context but for `currency: milestone?.currency` you are using optional chaining on the other hand for `amount: milestone.value` you are not. Which looks odd to me.